include/oneapi/dpl/internal/exclusive_scan_by_segment_impl.h

```diff
@@ -44,7 +45,7 @@ exclusive_scan_by_segment_impl(Policy&& policy, InputIterator1 first1, InputIter
 {
     const auto n = ::std::distance(first1, last1);
 
-    // Check for empty and single element ranges
+    // Check for empty element ranges
```

Comment:
Why this comment was changed? It's describes two next conditions, I think

Reply:
This was an unintended change that I did not realize was made. I will revert this back.